soccer/config/sim.xml

```diff
@@ -358,38 +355,40 @@
    <min value="1"/>
   </Health>
  </VisionFilter>
+ <Replanner>
+  <goalPosChangeThreshold value="0.025"/>
+  <goalVelChangeThreshold value="0.025"/>
+  <partialReplanLeadTime value="0.1"/>
+ </Replanner>
+ <LineKickPlanner>
+  <approachSpeed value="0.25"/>
+ </LineKickPlanner>
  <Capture>
+  <searchStartDist value="0"/>
+  <searchEndDist value="7"/>
+  <searchIncDist value="0.2"/>
+  <maxBallSpeedDirect value="0.1"/>
+  <maxApproachAngle value="0.3"/>
+  <maxOutrunBallSpeed value="0.7"/>
+  <ballContactAccelPercent value="0.2"/>
+  <ballSpeedPercentForDampen value="0"/>
+  <Collect>
+   <bufferDistBeforeContact value="0.08"/>
+   <bufferDistAfterContact value="0.08"/>
+   <targetChangeThreshold value="0.025"/>
+   <ballSpeedApproachDirectionCutoff value="0.1"/>
+   <touchDeltaSpeed value="0.15"/>
+   <controlAccelScalePercent value="0.8"/>
+   <distCutoffToControl value="0.0285"/>
+  </Collect>
   <Settle>
-   <ballSpeedPercentForDampen value="0"/>
-   <searchStartTime value="0.1"/>
-   <searchEndtime value="10"/>
-   <searchIncTime value="0.2"/>
-   <interceptBufferTime value="1"/>
+   <bufferTimeBeforeContact value="0.0"/>
    <targetPointGain value="0.5"/>
-   <ballVelGain value="0.9"/>
-   <maxAnglePathTargetChange value="20"/>
-   <maxBallVelForPathReset value="2"/>
-   <maxBallAngleForReset value="20"/>
```

Comment:
A bunch of these have been deleted reverting to their default value. At one point I set the defaults to my best config at the time, but it's been tuned since them I imagine